ui_testing/pages/testunits_page.py

```diff
@@ -53,7 +60,7 @@ def restore_selected_test_units(self):
         self.base_selenium.click(element='test_units:restore')
         self.confirm_popup()
 
-    def get_active_test_units(self):
+    def get_active_selected_test_units(self):
```

Comment:
I'm wondering did u change the calling of this method in the other test cases?

Reply:
no, it is most likely due to have old version while creating branch, I didn't change it.